### PR TITLE
RELEASE-542: Release PLF 5.0.0-RC06

### PR DESCRIPTION
**Pull request by mhbensghaier**

### CODE REVIEW

File: exo-platform/RELEASE-542.json [hunk 1]
```diff
@@ -0,0 +1,883 @@
...
+    "git_organization":"exoplatform",
+    "labels":"addons",
+    "maven_property_version":"addon.exo.answers.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/1.3.x",
+      "version":"1.3.0-RC06",
```
Comment (vsellier) [inline exo-platform/RELEASE-542.json]: According to [SWF-4221](https://jira.exoplatform.org/browse/SWF-4221) :
Stable branch -> ``stable/2.0.x``
Version -> ``2.0.0-RC06``

File: exo-platform/RELEASE-542.json [hunk 2]
```diff
@@ -0,0 +1,883 @@
...
+    "git_organization":"exoplatform",
+    "labels":"addons",
+    "maven_property_version":"addon.exo.cas.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/1.3.x",
+      "version":"1.3.0-RC06",
```
Comment (vsellier) [inline exo-platform/RELEASE-542.json]: According to [SWF-4221](https://jira.exoplatform.org/browse/SWF-4221) :
Stable branch -> ``stable/2.0.x``
Version -> ``2.0.0-RC06``

File: exo-platform/RELEASE-542.json [hunk 3]
```diff
@@ -0,0 +1,883 @@
...
+    "git_organization":"exoplatform",
+    "labels":"addons,trial",
+    "maven_property_version":"addon.exo.chat.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/1.6.x",
+      "version":"1.6.0-RC06",
```
Comment (vsellier) [inline exo-platform/RELEASE-542.json]: According to [SWF-4221](https://jira.exoplatform.org/browse/SWF-4221) :
Stable branch -> ``stable/2.0.x``
Version -> ``2.0.0-RC06``

File: exo-platform/RELEASE-542.json [hunk 4]
```diff
@@ -0,0 +1,883 @@
...
+    "git_organization":"exoplatform",
+    "labels":"addons",
+    "maven_property_version":"addon.exo.openam.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/1.3.x",
+      "version":"1.3.0-RC06",
```
Comment (vsellier) [inline exo-platform/RELEASE-542.json]: According to [SWF-4221](https://jira.exoplatform.org/browse/SWF-4221) :
Stable branch -> ``stable/2.0.x``
Version -> ``2.0.0-RC06``

File: exo-platform/RELEASE-542.json [hunk 5]
```diff
@@ -0,0 +1,883 @@
...
+    "git_organization":"exoplatform",
+    "labels":"addons,trial",
+    "maven_property_version":"addon.exo.remote-edit.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/1.3.x",
+      "version":"1.3.0-RC06",
```
Comment (vsellier) [inline exo-platform/RELEASE-542.json]: According to [SWF-4221](https://jira.exoplatform.org/browse/SWF-4221) :
Stable branch -> ``stable/2.0.x``
Version -> ``2.0.0-RC06``

File: exo-platform/RELEASE-542.json [hunk 6]
```diff
@@ -0,0 +1,883 @@
...
+    "git_organization":"exoplatform",
+    "labels":"addons",
+    "maven_property_version":"addon.exo.saml.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/1.3.x",
+      "version":"1.3.0-RC06",
```
Comment (vsellier) [inline exo-platform/RELEASE-542.json]: According to [SWF-4221](https://jira.exoplatform.org/browse/SWF-4221) :
Stable branch -> ``stable/2.0.x``
Version -> ``2.0.0-RC06``

File: exo-platform/RELEASE-542.json [hunk 7]
```diff
@@ -0,0 +1,883 @@
...
+    "git_organization":"exoplatform",
+    "labels":"addons",
+    "maven_property_version":"addon.exo.spnego.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/1.3.x",
+      "version":"1.3.0-RC06",
```
Comment (vsellier) [inline exo-platform/RELEASE-542.json]: According to [SWF-4221](https://jira.exoplatform.org/browse/SWF-4221) :
Stable branch -> ``stable/2.0.x``
Version -> ``2.0.0-RC06``

File: exo-platform/RELEASE-542.json [hunk 8]
```diff
@@ -0,0 +1,883 @@
...
+    "git_organization":"exoplatform",
+    "labels":"addons,trial",
+    "maven_property_version":"addon.exo.tasks.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/1.3.x",
+      "version":"1.3.0-RC06",
```
Comment (vsellier) [inline exo-platform/RELEASE-542.json]: According to [SWF-4221](https://jira.exoplatform.org/browse/SWF-4221) :
Stable branch -> ``stable/2.0.x``
Version -> ``2.0.0-RC06``

File: exo-platform/RELEASE-542.json [hunk 9]
```diff
@@ -0,0 +1,883 @@
...
+    "git_organization":"exoplatform",
+    "labels":"addons,trial",
+    "maven_property_version":"addon.exo.web-pack.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/1.2.x",
+      "version":"1.2.0-RC06",
```
Comment (vsellier) [inline exo-platform/RELEASE-542.json]: According to [SWF-4221](https://jira.exoplatform.org/browse/SWF-4221) :
Stable branch -> ``stable/2.0.x``
Version -> ``2.0.0-RC06``

File: exo-platform/RELEASE-542.json [hunk 10]
```diff
@@ -0,0 +1,883 @@
...
+      "current_snapshot_version":"5.0.x-SNAPSHOT",
+      "next_snapshot_version":"5.0.x-SNAPSHOT",
+      "patches":"",
+      "patches_after_release":"",
+      "nexus_host":"exoplatform",
+      "nexus_staging_profile":"addons"
+    }
+  },
+  {
+    "name":"crash-addon",
```
Comment (vsellier) [inline exo-platform/RELEASE-542.json]: This addon shoud not be present as it's now a community addon : [SWF-4222](https://jira.exoplatform.org/browse/SWF-4222)

Please remove it also on the list of project to release on RELEASE-542

File: exo-platform/RELEASE-542.json (hunk 9 above)
Comment (mhbensghaier) [reply exo-platform/RELEASE-542.json]: All requested chages are done

**Pull request merged**